graylog2-web-interface/src/routing/ApiRoutes.js

```diff
@@ -60,8 +61,8 @@ const ApiRoutes = {
     update: (decoratorId) => { return { url: `/search/decorators/${decoratorId}` }; },
   },
   DeflectorApiController: {
-    cycle: () => { return { url: '/cluster/deflector/cycle' }; },
-    list: () => { return { url: '/system/deflector' }; },
+    cycle: (indexSetId) => { return { url: `/cluster/deflector/${indexSetId}/cycle` }; },
```

Comment:
No, this needs to be `/cluster/..` to make this work on on-master nodes.

The `/cluster/deflector/{indexSetId}/cycle` endpoint was just missing.